src/pybind/mgr/dashboard/README.rst

```diff
@@ -1,40 +1,35 @@
 Dashboard and Administration Module for Ceph Manager
-=========================================================================
+====================================================
 
 Overview
 --------
 
-The original Ceph Manager Dashboard that was shipped with Ceph "Luminous"
+The original Ceph manager dashboard that was shipped with Ceph "Luminous"
 started out as a simple read-only view into various run-time information and
-performance data of a Ceph cluster.
+performance data of a Ceph cluster. It used a very simple architecture to
+achieve the original goal.
 
-However, there is a `growing demand <http://pad.ceph.com/p/mimic-dashboard>`_
-for adding more web-based management capabilities, to make it easier for
-administrators that prefer a WebUI over the command line.
+However, there was a growing demand for adding more web-based management
+capabilities, to make it easier to administer Ceph for users that prefer a WebUI
+over using the command line.
 
-This module is an ongoing project to add a native web based monitoring and
-administration application to Ceph Manager. It aims at becoming a successor of
-the existing dashboard, which provides read-only functionality and uses a
-simpler architecture to achieve the original goal.
+This new dashboard module is a replacement of the previous dashboard and an
```

Comment:
Suggestion: "[...] of the _previous_ one [...]" (instead of 2x "dashboard" in this sentence).

Reply:
Good idea, updated accordingly.